notebooks/json_blocks/WET/BLOCK-T413.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        "day": {
</pre><p>Is this <code>day_obs</code> in butler terms? I think you should just name this <code>day_obs</code> and the next one to <code>seq_num</code> to make clear what they are asking for. Would also rephrase the description to "day and year". Or do you actually mean the "n'th" day of the year? I would suggest not doing that and using <code>day_obs</code> instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>